src/circleMenu.lua

```diff
@@ -23,7 +14,7 @@ function CircleMenu.indexToAngle(index, division, startAngle)
 	return startAngle - math.pi * (-0.5 + ((index) * 2 - 1) / division)
 end
 
-function CircleMenu:draw(x, y, angle, size, strength, lables)
+function CircleMenu.draw(x, y, angle, size, strength, lables)
```

Comment:
As long as you're changing this line, can you fix the spelling of labels?

```suggestion
function CircleMenu.draw(x, y, angle, size, strength, labels)
```